chemprop/models/model.py

```diff
@@ -91,9 +90,9 @@ def __init__(
         self.X_d_transform = X_d_transform if X_d_transform is not None else nn.Identity()
 
         self.metrics = (
-            nn.ModuleList([*metrics, self.criterion])
+            nn.ModuleList([*metrics, self.criterion.clone()])
```

Comment:
Why clone here?

Reply:
From the [docs](https://lightning.ai/docs/torchmetrics/latest/pages/overview.html):
"Metrics contain internal states that keep track of the data seen so far. Do not mix metric states across training, validation and testing. It is highly recommended to re-initialize the metric per mode as shown in the examples above."
I use `.clone()` to get a new instance of the metric used for the training loss.